build-logic/src/main/kotlin/terasology-metrics.gradle.kts

```diff
@@ -25,13 +25,13 @@ dependencies {
     pmd("net.sourceforge.pmd:pmd-core:7.0.0-rc4")
     pmd("net.sourceforge.pmd:pmd-java:7.0.0-rc4")
 
-    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.11") {
+    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.12") {
         because("runtime: to configure logging during tests with logback.xml")
     }
     testRuntimeOnly("org.codehaus.janino:janino:3.1.7") {
         because("allows use of EvaluatorFilter in logback.xml")
     }
-    testRuntimeOnly("org.slf4j:jul-to-slf4j:1.7.36") {
+    testRuntimeOnly("org.slf4j:jul-to-slf4j:2.0.9") {
```

Comment:
@soloturn I think we should drop the upgrade from this PR entirely, not only for engine.
Thus, we can update it for all parts simultaneously.

Reply:
i always agree with your judgement. ok, most of the times :) but here i do not agree - using different jars or versions for testing must be possible, otherwise something is seriously wrong in the dependency tree.

Reply:
@soloturn I'm not saying that it isn't possible to use different versions :smile: 
I'm saying, if we noticed issues with slf4j v2, we should consider sticking with 1.7 for our tests as well until we figured out the issues with v2 at which point we can update it for engine and engine tests.

Reply:
@jdrueckert we know exaclty where and why it breaks, as it is in the hs_err_pid65260.log, in #5182. engine contains code relying on the old version, RegexFilterAction. jvm does not find the old binding, and crashes. crash is imo a jvm issue, but for terasology not important, broken is broken. so we do not upgrade until the old piece of code is fixed or removed. that does not mean we should block everything else.